Assets/PatchKit Patcher/Scripts/Patcher.cs

```diff
@@ -392,6 +392,7 @@ private void ThreadExecution(CancellationToken cancellationToken)
                     try
                     {
                         LauncherUtilities.ExecuteLauncher();
+                        return;
                     }
                     catch (ApplicationException)
```

Comment:
I think you should also add return in this catch.

Reply:
Done